opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -90,18 +95,20 @@ public TracingFilter() {
      * @param tracer
      */
     public TracingFilter(Tracer tracer) {
-        this(tracer, Collections.singletonList(ServletFilterSpanDecorator.STANDARD_TAGS));
+        this(tracer, Collections.singletonList(ServletFilterSpanDecorator.STANDARD_TAGS), null);
     }
 
     /**
      *
      * @param tracer tracer
      * @param spanDecorators decorators
+     * @param skipPattern null or pattern to exclude certain paths from tracing e.g. "/health"
      */
-    public TracingFilter(Tracer tracer, List<ServletFilterSpanDecorator> spanDecorators) {
+    public TracingFilter(Tracer tracer, List<ServletFilterSpanDecorator> spanDecorators, Pattern skipPattern) {
```

Comment:
This is also a good reason to include the rules with the framework integration, so they can be tested in conjunction with the specific version - but wanted to avoid that until the in-process context propagation stuff had settled.